apps/ecs/behaviours/send-email-notification.js

```diff
@@ -5,184 +5,174 @@ const notifyClient = new NotifyClient(notifyKey);
-    worker_email: req.sessionModel.get('before-1988-employer-email') ?? '',
-    has_ho_ref_number: req.sessionModel.get('steps').includes('/reference-number') ? 'yes' : 'no',
-    ho_reference_number: req.sessionModel.get('worker-reference-number')?.toUpperCase() ?? ''
+  getLabel = (fieldKey, fieldValue) => {
+    if (fieldValue) {
+      return translation[fieldKey].options[fieldValue].label;
+    }
+    return '';
   };
 
-  const dynamicProps = {};
-
-  if (recipientType === 'user') {
-    dynamicProps.business_address = req.sessionModel.get('businessAddress');
-    dynamicProps.worker_address = req.sessionModel.get('workerAddress') ?? req.sessionModel.get('workerUkAddress');
-  }
-
-  if (recipientType === 'business') {
-    dynamicProps.business_address_line_1 = req.sessionModel.get('business-address-line-1');
-    dynamicProps.business_address_line_2 = req.sessionModel.get('business-address-line-2') ?? '';
-    dynamicProps.business_address_town_city = req.sessionModel.get('business-town-city');
-    dynamicProps.business_address_postcode = req.sessionModel.get('business-postcode');
-
-    if (req.sessionModel.get('workerAddress')) {
-      dynamicProps.worker_address_line_1 = req.sessionModel.get('worker-address-line-1');
-      dynamicProps.worker_address_line_2 = req.sessionModel.get('worker-address-line-2') ?? '';
-      dynamicProps.worker_address_town_city = req.sessionModel.get('worker-town-or-city');
-      dynamicProps.worker_address_postcode = req.sessionModel.get('worker-zipcode') ?? '';
+  /* eslint-disable max-len */
+  documentTypeMap = req => {
+    if (req.sessionModel.get('use-digital-right-to-work') === 'yes') {
+      return 'or unable to use Digital Right to Work service due to technical error';
+    } else if (req.sessionModel.get('worker-applied-eu-settlement-scheme') === 'yes-certificate-of-application') {
+      return 'a Certificate of Application (COA) issued following submission of an application under the EU Settlement Scheme (EUSS)';
+    } else if (req.sessionModel.get('worker-applied-eu-settlement-scheme') === 'yes-evidence-of-settled-status') {
+      return 'evidence to demonstrate the individual has settled or pre-settled status under the EU Settlement Scheme (EUSS) issued in a Crown Dependency';
+    } else if (req.sessionModel.get('worker-applied-eu-settlement-scheme') === 'no-frontier-worker') {
+      return 'or is a Frontier Worker in the UK';
+    } else if (req.sessionModel.get('worker-has-arc-card') === 'yes') {
+      return 'an Application Registration Card (ARC) for an asylum seeker stating that the holder is allowed to work';
+    } else if (req.sessionModel.get('worker-have-ongoing-appeal') === 'yes') {
+      return 'an ongoing application or appeal for leave to remain in the UK';
+    } else if (req.sessionModel.get('worker-applied-for-settlement-protection') === 'yes') {
+      return 'Settlement Protection';
```

Comment:
there is no need for `else if` because you have a return in every statement, removing else if will improve readability